src/components/mapPageComponents/mapContainer/ducks/blockReducer.ts

```diff
@@ -0,0 +1,39 @@
+import { BlockGeoData, BlockGeoDataReducerState } from './types';
+import {
+  ASYNC_REQUEST_FAILED_ACTION,
+  ASYNC_REQUEST_LOADED_ACTION,
+  ASYNC_REQUEST_LOADING_ACTION,
+  AsyncRequestNotStarted,
+  generateAsyncRequestReducer,
+} from '../../../../utils/asyncRequest';
+import { blockGeoData } from './actions';
+import { C4CAction } from '../../../../store';
+
+export const initialBlockGeoDataState: BlockGeoDataReducerState = {
+  blockGeoData: AsyncRequestNotStarted<BlockGeoData, any>(),
+};
```

Comment:
I think we can probably combine the reducers as well and just maintain two separate fields in the single reducer state.